modules/migration-mtc-release-notes-1-7-15.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-15_{context}"]
+= {mtc-full} 1.7.15 release notes
+
+[id="resolved-issues-1-7-15_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.CVE-2024-24786: A flaw was found in Golang's protobuf module, where the unmarshal function can enter an infinite loop
+
+A flaw was found in the `protojson.Unmarshal` function, which could enter an infinite loop when unmarshaling certain forms of invalid JSON. This condition could occur when unmarshaling into a message that contained a `google.protobuf.Any` value or when the `UnmarshalOptions.DiscardUnknown` option was set.
```

Comment:
"which" is typically used for non-restrictive clauses but this statement seems essential· How about:

> A flaw was found in the `protojson.Unmarshal` function that could cause the function to enter an infinite loop when unmarshaling certain forms of invalid JSON messages. This condition could occur when unmarshaling into a message that contained a `google.protobuf.Any` value or when the `UnmarshalOptions.DiscardUnknown` option was set in a JSON-formatted message.